ledger/participant-integration-api/src/main/scala/platform/localstore/PersistentPartyRecordStore.scala

```diff
@@ -289,4 +289,13 @@ class PersistentPartyRecordStore(
     case Right(value) => value
     case Left(_) => Set.empty
   }
+
+  override def filterExistingParties(parties: Set[Party])(implicit
+      loggingContext: LoggingContext
+  ): Future[Set[Party]] = inTransaction(_.partiesExist) { implicit connection =>
+    Right(backend.filterExistingParties(parties)(connection))
+  }.map {
+    case Right(value) => value
+    case Left(_) => Set.empty
```

Comment:
Why is it ok to ignore the left? Could this hide some underlying failure and produce misleading errors/results elsewhere?

Reply:
because `Left` is not available here as an option, you see on line 296 it's always `Right` :)

Reply:
alternatively I could have a special `inTransaction` method which would not be expecting `Either`, wdyt?

Reply:
Oh I see it now. Up to you.
(Or leave a comment `// We're always Right. See above` ;))